File: src/main/java/org/cimsbioko/fragment/navigate/detail/DefaultDetailFragment.java
package org.cimsbioko.fragment.navigate.detail;

import org.cimsbioko.R;
import org.cimsbioko.data.DataWrapper;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

public class DefaultDetailFragment extends DetailFragment {

	@Override
	public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
		return inflater.inflate(R.layout.default_detail_fragment, container, false);
	}

	@Override
	public void setUpDetails(DataWrapper data) {
	}
}

File: src/main/kotlin/org/cimsbioko/fragment/navigate/detail/DefaultDetailFragment.kt
package org.cimsbioko.fragment.navigate.detail

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import org.cimsbioko.R
import org.cimsbioko.data.DataWrapper

class DefaultDetailFragment : DetailFragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.default_detail_fragment, container, false)
    }

    override fun setUpDetails(data: DataWrapper?) {}
}